src/main/java/org/example/BankAccount.java

```diff
@@ -0,0 +1,30 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class BankAccount {
+    private final String own;
+    private final String accountNumber;
+    private double total;
+
+    public BankAccount(String own, String accountNumber){
+        this.own=own;
+        this.accountNumber=accountNumber;
+        this.total = 1000;
+    }
+    public void withdraw(){
+        Scanner scanner = new Scanner(System.in);
```

Comment:
**Resource leak** "scanner" is never closed.

Try to open just one Scanner object and send it as a parameter when you need it